### DISCUSSION TITLE
No puedo enviar las respuestas al fomulario

### DISCUSSION THREAD

**Discussion by erick129n** [LaunchX-InnovaccionVirtual/CursoIntroPython#333] [Q&A] [answered]
Estaba ya solo poniendo el link de mi repositorio y cuando le di en enviar ya no me dejaba enviar las respuestas, quise abrirlo de nuevo pero ahora me dice que ya no se aceptan respuestas al formulario, URGENTE PORFAVOR.
Ya que Fernanda Ochoa, dejo muy en claro que el formulario se cierra el lunes 21/02/2022, a la 1 de la tarde. 
Espero que puedan responder y ayudarme porfavor.

**Comment by Gustavo-Valdez** [comment]
Me pasa lo mismo solo que ella dijo que seria a las 8 del dia de mañana :((

**Comment by enrique3240** [comment, marked as answer]
Vuelvan a intentar, al parecer nuevamente está habilitado. Acabo de enviar las respuestas del formulario sin problemas, después de que me apareciera lo mismo que a ustedes.

**Reply by ericmanzanillac** [reply]
Si, me acaba de volver a dejar. Solo que me dice que no puedo responderlo, según yo ya está iniciada la sesión.

**Comment by GreciaCardona** [comment]
A mi tampoco me dejo, estaba revisando mis datos y al darle enviar me decía que ya no se aceptaban respuestas y eran justamente las 8 :'o u_u
